lib/workshops.dart

```diff
@@ -180,6 +180,7 @@ class WorkshopUi extends EditorUi {
       if (solutionShownThisStep) {
         // Enable the show solution button
         showSolutionButton.disabled = false;
+        showSolutionButton.textColor = 'blue';
```

Comment:
This doesn't match the original color:
![Screenshot 2023-05-10 at 1 45 40 PM](https://github.com/dart-lang/dart-pad/assets/1145719/834900c6-90c1-48ce-bfd9-71d3e032ab59)

![Screenshot 2023-05-10 at 1 45 33 PM](https://github.com/dart-lang/dart-pad/assets/1145719/f9b1cc2c-2c87-47f8-a438-d0a5697e0c53)